### PR TITLE
chore: removing redundant docker-entrypoint

### PR DESCRIPTION
**Pull request by ofekisr**
### SUMMARY
Please review the docker-entrypoint.sh, it is redundant it does not really do something, 
so you can set the default command in the image to running the server and can always override the CMD
and we earn preventing code reusing

note- think of the chart version, should it be incremented to 0.5.0 or 1.0.0 ? 
the chart must use the new image

### TESTING INSTRUCTIONS
let it build and run

### CODE REVIEW

**Review by craig-rueda — APPROVED** [review]
LGTM

**Comment by AndreyRomanchev** [comment]
helm chart uses tag latest (not a great thing), so old superset helm installations will fail with error
```/bin/sh: 1: /usr/bin/docker-entrypoint.sh: not found```
if the new image will be downloaded for some reason (pod restart or schedule to new node for example)

I upgraded helm chart from 0.3.10 to 0.5.0, this resolved the issue.

File: docker/docker-bootstrap.sh [hunk 1]
```diff
@@ -47,5 +47,5 @@ elif [[ "${1}" == "app" ]]; then
   flask run -p 8088 --with-threads --reload --debugger --host=0.0.0.0
 elif [[ "${1}" == "app-gunicorn" ]]; then
   echo "Starting web app..."
-  /app/docker/docker-entrypoint.sh
+  /usr/bin/run-server.sh
```
Comment (d9k) [inline docker/docker-bootstrap.sh:50]: Having trouble starting docker composer due to this fix:

```
/app/docker/docker-bootstrap.sh: line 50: /usr/bin/run-server.sh: No such file or directory
```

Comment (craig-rueda) [reply docker/docker-bootstrap.sh:50]: Try running `docker-compose pull` first

Comment (msxiehui) [reply docker/docker-bootstrap.sh:50]: > 

macOS 
 /usr/bin/run-server.sh   ->  sh /app/docker/run-server.sh

**Comment by layoaster** [comment]
the [image](https://hub.docker.com/layers/apache/superset/1.3.2/images/sha256-459c653ce3db6931bd7f9911e2fbfb93f5385f815cddf63a897fcbc6d1d73f33?context=explore) for Superset version 1.3.2 is not updated with these changes.

My values.yaml:
```yaml
image:
  repository: apache/superset
  tag: "1.3.2"
  pullPolicy: IfNotPresent
```

What official image tag should I use to get the latest version working with this chart version ??

**Pull request merged**